ecosystem/sep-0010.md

```diff
@@ -85,8 +86,7 @@ In order for browsers-based wallets to validate the CORS headers, as [specified
 
 ### Challenge
 
-This endpoint must respond with a Stellar transaction signed by the server that has an invalid sequence number (0) and thus cannot be executed on the Stellar network. The client can then sign the transaction using standard Stellar libraries and submit it to [`token`](#token) endpoint to prove that they control their account. This approach is compatible with hardware wallets such as Ledger. The client must also verify the server's signature to be sure the challenge is signed by the `SIGNING_KEY`, and the home domain in the challenge is the home domain from the discovery flow.
-
+This endpoint must respond with a Stellar transaction signed by the ``SIGNING_KEY`` from the **Home Domain**'s [`stellar.toml`](sep-0001.md) file that has an invalid sequence number (0) and thus cannot be executed on the Stellar network. The **Client Application** can then sign the transaction using standard Stellar libraries and submit it to [`token`](#token) endpoint to prove that it controls the **Authenticating Account**. This approach is compatible with hardware wallets such as Ledger. The **Client Application** must also verify the home domain in the first operation of the challenge is the **Home Domain**.
```

Comment:
❗️In the last sentence of this section it called out that the client must verify the servers signature. That's been removed, but it seems unnecessary to me to remove it if this change is scoped to clarifying the participants. We kept the statement to verify the home domain, which seems like we're playing more importance on that when everything that needs verifying is important.